conference.js

```diff
@@ -219,7 +219,13 @@ function maybeRedirectToWelcomePage(options) {
     if (config.enableWelcomePage) {
         setTimeout(() => {
             APP.settings.setWelcomePageEnabled(true);
-            window.location.pathname = "/";
+
+            // XXX To support a deployment in a sub-directory, assume that the
+            // room (name) is the last non-directory component of the path
+            // (name). We already had the assumption before changing / to ./
+            // here because we are navigating to close.html, close2.html (above)
+            // and authError.html.
+            window.location.pathname = "./";
```

Comment:
The source code comment states that `./` is about supporting a context root. I'll try to think of more details but I'll first have to implement an alternative because @guusdk says `./` makes no difference in his tests.